Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/testresult/pending/SubmissionTestResultPendingFragment.kt

```diff
@@ -60,11 +62,22 @@ class SubmissionTestResultPendingFragment : Fragment(R.layout.fragment_submissio
             }
         }
 
+        binding.apply {
```

Comment:
Its a bit hard to figure out which view is displayed in which case 😅.

what about keeping it simple like

```
when (viewModel.testType) {
    CoronaTest.Type.PCR -> {
        testResultPendingStepsContactDiaryResult.isVisible = false
        testResultPendingStepsWaitingPcrResult.isVisible = true
        // and so on
    }
    CoronaTest.Type.RAPID_ANTIGEN -> {
        testResultPendingStepsContactDiaryResult.isVisible = true
        testResultPendingStepsWaitingPcrResult.isVisible = false
        // and so on
   }
}
```